packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1264,6 +1264,30 @@ class JsonViewer extends StatefulWidget {
   State<JsonViewer> createState() => _JsonViewerState();
 }
 
+class TextViewer extends StatelessWidget {
+  const TextViewer({
+    super.key,
+    required this.text,
+  });
+  final String text;
+  static const _maxLength = 65536; //2^16
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final String displayText;
+    // Limit the length of the displayed text to _MAX_LENGTH
```

Comment:
We should reference the issue here.

Reply:
I'll reference it in the network code where we reference the TextViews.